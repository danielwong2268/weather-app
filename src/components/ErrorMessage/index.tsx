import { RED_1 } from 'src/constants/colors';
import styled from 'react-emotion';

const ErrorMessage = styled('div')`
  color: ${RED_1};
`;

export default ErrorMessage