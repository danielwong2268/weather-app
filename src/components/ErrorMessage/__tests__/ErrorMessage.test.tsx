import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ErrorMessage from '..';

describe('<ErrorMessage />', () => {
  it('Renders with an error message', () => {
    const renderer = ShallowRenderer.createRenderer();
    renderer.render(<ErrorMessage>my error message</ErrorMessage>);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});