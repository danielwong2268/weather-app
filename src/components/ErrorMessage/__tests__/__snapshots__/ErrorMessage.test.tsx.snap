// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> Renders with an error message 1`] = `
<div
  className="css-1kpr6lr"
>
  my error message
</div>
`;
